stellarphot/core.py

```diff
@@ -106,7 +108,9 @@ class BaseEnhancedTable(QTable):
         This is a dictionary where each key is a required table column name
         and the value corresponding to each key is the required dtype
         (can be None).  This is used to check the format of the input data
-        table.
+        table.  Columns will be output in the order of the keys in the dictionary
+        with any additional columns tacked on the end. This requires python >=3.7
```

Comment:
We can drop this because we require Python >=3.10, so the old case can't come up.

```suggestion
        with any additional columns tacked on the end. 
```